chartbeat.php

```diff
@@ -30,6 +30,16 @@ function chartbeat_menu() {
 }
 add_action('admin_menu', 'chartbeat_menu');
 
+function chartbeat_accountid_error() {
+	$class = 'notice notice-error';
+	$message = 'You need to set your Chartbeat <a href="'.admin_url( 'options-general.php?page=chartbeat-options' ).'">Account ID</a> in the Chartbeat options page';
+
+	printf( '<div class="%1$s"><p>%2$s</p></div>', $class, $message ); 
+}
+if (!get_option('chartbeat_userid') ) {
```

Comment:
This should probably be moved to a callback on the `admin_init` hook. Right now, it fires on every single pageload.